pyphare/pyphare/pharein/simulation.py

```
@@ -395,6 +395,10 @@ def check_restart_options(**kwargs):
         if "mode" not in restart_options:
             raise ValueError (f"Restart mode not set, valid modes are {valid_modes}")
 
+        if "dir" in restart_options:
+            while restart_options["dir"][-1] == "/": # samrai can't handle this
```

maybe should be 

```
while len(restart_options["dir"]) > 1 restart_options["dir"][-1] == "/
```

?

---

actually 

```
while "//" in restart_options["dir"]: restart_options["dir"] = restart_options["dir"].replace("//", "/")
```

---

or maybe just raise and say it's invalid

---

"aaa/bbb///".rstrip("/")

---

IMO we should also do other checks such as permissions (https://www.tutorialspoint.com/How-to-check-the-permissions-of-a-directory-using-Python), other things?

---

check there's some space left on the disk maybe

---

probably not straightforward with the quotas